### PR TITLE
Counterspell Abu Dhabi

### PR DESCRIPTION
**Pull request by akulsaju**

### CODE REVIEW

**Review by samdev-7 — CHANGES_REQUESTED** [review]
You need to make a copy of the example components. I'd check out how other cities do it in the `components` folder. Also, use `AbuDhabi` as the folder name, do not use spaces.

**Comment by akulsaju** [comment]
@samdev-7 done

File: pages/abudhabi.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+import Head from "next/head";
+
+// TODO: Change `ExampleCity` to the name of your city. This should match the folder you copied!
+import Hero from "../components/Abu Dhabi/Hero";
+import About from "../components/Abu Dhabi/About";
+import Guilds from "../components/Abu Dhabi/Guilds";
+import Steps from "../components/Abu Dhabi/Steps";
+import Locations from "../components/Abu Dhabi/Locations";
+import Faq from "../components/Abu Dhabi/Faq";
+import Footer from "../components/Abu Dhabi/Footer";
```
Comment (samdev-7) [inline pages/abudhabi.js:37]: These imports are wrong. You'll need to remove the space.

**Review by samdev-7 — CHANGES_REQUESTED** [review]

File: pages/abudhabi.js (hunk 1 above)
Comment (akulsaju) [reply pages/abudhabi.js]: @samdev-7 done

File: pages/abudhabi.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+import Hero from "../components/AbuDhabi/Hero";
+import About from "../components/AbuDhabi/About";
+import Guilds from "../components/AbuDhabi/Guilds";
+import Steps from "../components/AbuDhabi/Steps";
+import Locations from "../components/AbuDhabi/Locations";
+import Faq from "../components/AbuDhabi/Faq";
+import Footer from "../components/AbuDhabi/Footer";
+
+// TODO: Change `ExampleCity` to the name of your city
+export default function Abu Dhabi() {
```
Comment (samdev-7) [inline pages/abudhabi.js:40]: Same with here. I'd recommend that you try testing your changes locally with `yarn` and `yarn dev`!

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by akulsaju** [comment]
Done

**Comment by samdev-7** [comment]
I don't see any changes. Make sure you’ve pushed the changes.

File: .github/workflows/nextjs.yml
Comment (samdev-7) [inline .github/workflows/nextjs.yml:1]: You don't need to create this file!

File: pages/abudhabi.js (hunk 2 above)
Comment (samdev-7) [reply pages/abudhabi.js:40]: Please change this to AbuDhabi() 
The space is causing an error.

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by akulsaju** [comment]
I had fixed it before but didn't commit

File: .github/workflows/nextjs.yml
Comment (samdev-7) [reply .github/workflows/nextjs.yml:1]: Please remove this file!

**Comment by samdev-7** [comment]
Remember to push/sync after you commit! I don't see any changes.

**Comment by akulsaju** [comment]
@samdev-7 done

**Review by samdev-7 — APPROVED** [review]

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Seems like the code is failing. You need to capitalize the file names in components/AbuDhabi.

**Comment by akulsaju** [comment]
@samdev-7 updated the file names and made a few changes in Schedule.js

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/abudhabi**](https://counterspell-gvj2vdnhw.hackclub.dev/abudhabi) |
|:-:|
|<a href="https://counterspell-gvj2vdnhw.hackclub.dev/abudhabi">Screenshot of /abudhabi</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/v54LT5lMyok_yleMg1zUxF-OVF2GT5osxJSspsrVS3N2mqtUrVTjGASHrSEtYsrN2qaHuuwwW8VvvoPwQrT84Q.png">(view full size)</a> |







Commit ff533d566117518d5a1ba64a262df7ebb43f4214 (https://counterspell-gvj2vdnhw.hackclub.dev).

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**